doc/logging.md

```diff
@@ -0,0 +1,17 @@
+# How does Happy test log work
+
+## Types of Happy logs
+  * Happy debug logs — Happy will run as test coordination and orchestrator, facilitating the activities of the virtual test device nodes. Happy debug logs will have logs from Happy tool only.
+
+  * Happy device logs — Happy will host virtual network which has multiple virtual devices, Happy device logs will have logs for software running on the virtual device, OpenWeave is one of the software example, so device logs will have OpenWeave logs.
+
+## Where Happy logs are stored
+
+Both Happy host and device logs location are processed in `happy/Driver.py` as below:
```

Comment:
"...are processed in `happy/Driver.py` as below" seems like an implementation detail that should be elided here.

Reply:
removed that line